app/serializer.py

```diff
@@ -0,0 +1,35 @@
+import json
+import xml.etree.ElementTree as ET
+from abc import ABC, abstractmethod
+
+from app.book import Book
+
+
+class AbstractSerializer(ABC):
+    @abstractmethod
+    def serialize(self) -> None:
+        pass
+
+
+class JSONSerializer(AbstractSerializer):
+
```

Comment:
There is no need to add a blank line after the class definition and before the first methods that are present inside. Remove such blank lines in all classes in the code